src/RelayEnvironment.ts

```diff
@@ -0,0 +1,9 @@
+import { Environment, Network, RecordSource, Store } from 'relay-runtime'
```

Comment:
wonder if all this should live under `src/relay` rather than spread out

Reply:
<img width="478" alt="Screen Shot 2022-08-16 at 2 52 55 PM" src="https://user-images.githubusercontent.com/4899429/184957649-54b69942-20f5-41a6-ab89-4af9b13dcfe4.png">